tests/CMakeLists.txt

```diff
@@ -85,3 +85,32 @@ if(NOT CMAKE_CROSSCOMPILING)
 else()
     gtest_add_tests(TARGET test_regression)
 endif()
+
+# When APPLE_STATIC_LOADER is ON installation is disabled
+if (APPLE_STATIC_LOADER)
+    return()
+endif()
+
+# Test installation
+set(test_install_dir "${CMAKE_CURRENT_BINARY_DIR}/install")
+add_test(NAME integration.install
+    COMMAND ${CMAKE_COMMAND} --install ${PROJECT_BINARY_DIR} --prefix ${test_install_dir} --config $<CONFIG>
+)
+
+# find_package testing currently doesn't work well under cross-compilation scenarios.
+if (CMAKE_CROSSCOMPILING OR
+    NOT CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
This 1 line was added to make 32 bit builds work without issue. I'm not a fan and it could potentially be handled more elegantly.